runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkCombineFn.java

```diff
@@ -722,6 +722,8 @@ public PipelineOptions getPipelineOptions() {
           WindowedAccumulator.create(this, toValue, windowingStrategy, windowComparator);
       accumulator.add(value, this);
       return accumulator;
+    } catch (RuntimeException ex) {
+      throw ex;
```

Comment:
This is exactly what fixes `ViewTest.testNonSingletonSideInput`. Some internal runners-core stuff throw `RuntimeException`s like `IllegalArgumentException` and the tests expect those, so we should not be re wrapping those in a different kind.